README.md

```diff
@@ -1,2 +1,21 @@
-# hw01
-Repository for homework 01
+![Image of Me](GitHubPhoto.jpg)
+
+# Eitan's Homework #1: A Brief Biography
+
+My story begins in Brooklyn on April 2nd, 1998. I went to to Poly Prep Country Day School in Brooklyn where I was primarily engaged in high school debate. Now a Junior at the University of Chicago majoring in *Political Science* and *Psychology*. 
+
+### Career Goals
+
+I have changed my decision on what I would like to do several times in my college career. I started out sure that I wanted to go to law school, then flirted with the idea of consulting or working at a startup, and am now back to thinking I would like to go to law school at some point. My linkedin is attached [here](https://www.linkedin.com/in/eitan-ezra-35b53b165/) if you'd like a visual representation of what I've been doing with my time in the past. 
+
+### My Interests
+
+My life is not just school work. in my free time I like to:
+
```

Comment:
You don't need to assign the numbers yourself.

```
1. Read primarily about international relations.
1. Play basketball even though I'm still *very* bad.
1. Watch **mindless** Netflix shows about solving crimes.
```

renders as

1. Read primarily about international relations.
1. Play basketball even though I'm still *very* bad.
1. Watch **mindless** Netflix shows about solving crimes.